DistributionPackages/Sfi.Kateheo/Resources/Private/Fusion/Objects/DefaultLayout.fusion

```diff
@@ -6,16 +6,20 @@ prototype(Sfi.Kateheo:DefaultLayout) < prototype(Neos.Fusion:Component) {
 	renderer = afx`
 		<a class="SkipToContent" href="#maincontent">Перейти к основному содержимому</a>
 		<div class="js-MobileNav">
-			<div class="MobileNav-Content">
-				<div class="fixedWidth">
-					<Sfi.Kateheo:Header/>
-				</div>
-				<div class={props.fixedWidth && 'fixedWidth'}>
-					<div id="maincontent">
-						{props.main}
+			<div class="MobileNav-Content" style="display: flex; min-height: 100vh; flex-direction: column;">
```

Comment:
Just curious, is `min-height` really necessary here?

Reply:
Yes, without it - not working.